transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -470,7 +517,8 @@ impl SignatureRctBulletproofs {
     /// * `output_values_and_blindings` - Value and blinding for each output
     ///   amount commitment.
     /// * `fee` - Value of the implicit fee output.
-    /// * `token id` - This determines the pedersen generator for commitments
+    /// * `signer` - The ring signer entity (with spend private key)
+    /// * `rng` -
     pub fn sign<CSPRNG: RngCore + CryptoRng, S: RingSigner + ?Sized>(
```

Comment:
I actually would like to remove this function at this point, and just have `SigningData.sign`, but for now I'll leave it.